docs/basic-docs/atomicdex-api-legacy/setprice.md

```diff
@@ -50,6 +50,8 @@ To prevent a user from making trades in which the transaction fees may end up co
 | result.conf_settings.base_nota  | bool             | whether dPoW notarization is required for base coin atomic swap transaction                               |
 | result.conf_settings.rel_confs  | number           | number of required blockchain confirmations for rel coin atomic swap transaction                          |
 | result.conf_settings.rel_nota   | bool             | whether dPoW notarization is required for rel coin atomic swap transaction                                |
+| result.base_orderbook_ticker            | string                     | the ticker of the base currency (if alternative addresses are active (e.g. legacy/segwit) in `coins` file)|
```

Comment:
`orderbook_ticker` use case might be beyond alternative addresses scope. We can possibly use it to have the same orderbook for coins with different protocols (USDC-ERC20, USDC-BEP20, etc.). Could you please change it to something like `orderbook_ticker is configured for the base/rel coin in coins file`?
Please also indicate explicitly that it can have a null value.